sdk/identity/azure-identity/tests/test_aad_client.py

```diff
@@ -44,7 +44,7 @@ def test_error_reporting():
         assert transport.send.call_count == 1
         transport.send.reset_mock()
 
-
+@pytest.mark.skip(reason="Adding body to HttpResponseError str. Not an issue bc we don't automatically log errors")
```

Comment:
Is this anything in these tests that should be preserved outside of thigs new str behaviour?

Reply:
Second. Seems like we changed our expected behavior. Maybe we should remove the tests?

Reply:
I checked with @mccoyp and @johanste and they're ok with skipping the tests (at least for now). The test basically mocks a response with secrets, and wants to make sure we don't expose the mock secrets in the str of error response.

@xiangyan99 maybe we can delete, but I think for now we can just skip, just in case